videostream.js

```diff
@@ -28,6 +25,10 @@ function VideoStream (file, mediaElem, opts) {
 				track.muxed.destroy()
 				track.muxed = muxed[i]
 				track.mediaSource = self._elemWrapper.createWriteStream(track.mediaSource)
+
+				track.mediaSource.on('error', function (err) {
```

Comment:
You should be adding this handler in the 'ready' case too, which handles the first seek.

Actually, just move this right before https://github.com/jhiesey/videostream/pull/25/files#diff-5ef112e381cabd910d268c65fafe5efcR58 and you should be set!